modules/pulsar/out/production/resources/proxy.conf

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
this file seems to be committed by accident ( `modules/pulsar/out/production/` folder)